tests/bulk_create/models.py

```diff
@@ -51,3 +53,17 @@ class TwoFields(models.Model):
 
 class NoFields(models.Model):
     pass
+
+
+class NullableFields(models.Model):
```

Comment:
I guess all fields should be tested to prevent a line in `BulkInsertMapper` from being inadvertently removed.